SYCL/ESIMD/noinline_bypointers_vadd.cpp

```diff
@@ -0,0 +1,112 @@
+//==------ noinline_bypointers_vadd.cpp  - DPC++ ESIMD on-device test ------==//
```

Comment:
Please move to https://github.com/intel/llvm-test-suite/tree/intel/SYCL/ESIMD/regression

Reply:
Done